### ISSUE TITLE
Plugin CPU usage unusually high even when idle

### ISSUE THREAD

**Issue by micpp42** [Ertego/gog-galaxy-itch.io#29]
I've noticed that the python process attached to this plugin has a tendency for me to just sit there while everything is idle using about 9% CPU, even when not otherwise being used. I'm not sure if this is related to the tendency of the plugin to continually be writing and filling up log files much faster than most of the other plugins, but that might be a starting point.

[plugin-itch-fd0effe6-77d9-42bc-999d-a5a5c870c2a4.log](https://github.com/Ertego/gog-galaxy-itch.io/files/4789850/plugin-itch-fd0effe6-77d9-42bc-999d-a5a5c870c2a4.log)

**Comment by lele-blue** [comment]
I think this might be related with #23, witch will be resolved with #6.
I'll leave this issue open until #6 is implemented and test if it helps

**Comment by micpp42** [comment]
Yeah, I'd say it looks related to 23, since I was looking at some of my logs and it seems like it starts scanning right away after the last scan ends - like continuously instead of just e.g once every so often
